module/user-roles/helper.js

```diff
@@ -87,4 +87,16 @@ module.exports = class UserRolesHelper {
     });
   }
 
+      
+  static findOne(query){
```

Comment:
if you doing find then function name should match @praveenKDass

Reply:
resolved